docs/algorithms/castor/special-features/hvdc.md

```diff
@@ -0,0 +1,110 @@
+# HVDC
+
+## Definition
+
+HVDC lines are modeled using [HvdcLine of the IIDM PowSyBl network model](inv:powsyblcore:*:*#hvdc-line), and an HVDC Range Action will always point to an HVDC line.
+
+An HVDC line can operate in **two modes**:
+
+1. **Fixed Setpoint Mode**  
+   In this mode, the power flow on the HVDC line is directly controlled via its **active power setpoint**.
+
+2. **AC Emulation Mode**  
+   The HVDC line operates in AC Emulation mode when the extension [**angle-droop active power control**](inv:powsyblecore:*:*#hvdc-angle-droop-active-power-control) is enabled. In this mode, the flow is determined by the **phase difference**:  
+   \[
+   P = P_0 \cdot k (\phi_1 - \phi_2)
+   \]  
+   and **ignores the active power setpoint**.
+
+**Implication:**
+
+An HVDC Range Action moves the active setpoint field of the HVDC line. When a line is in AC Emulation mode, applying an **HVDC Range Action** has no effect because the line cannot be directly controlled via its active power setpoint.  
+For the same reason, attempting to define a sensitivity for such a range action will result in an **OLF error**. See [Open Load Flow documentation](https://powsybl.readthedocs.io/projects/powsybl-open-loadflow/en/latest/sensitivity/getting_started.html).
+
+---
+
+## HVDC Range Action in Each Process
+
+### Italy Nord Process (CSE Crac)
+
+The HVDC line is disconnected and replaced by two injections, one on each side of the line, with opposite keys of 1 and -1, as illustrated in the following diagram.
```

Comment:
I also wrote this comment in the CSE file below, but you have to fix this twice:

the HVDC line is not *disconnected*, the reality is that an HVDC line cannot be modeled in UCTE format, so an equivalent model is used: the HVDC is represented through two injections (and other elements, cf GridCapa doc), this is why the HVDC range action is modelled by an Injection Range actions.